mysql-test/t/bug90351.test

```diff
@@ -0,0 +1,27 @@
+#
+# Debug build tests for INFORMATION_SCHEMA.GLOBAL_TEMPORARY_TABLES
+#
+--source include/have_innodb.inc
+--source include/have_debug.inc
+--source include/have_debug_sync.inc
+
+connect (con1,localhost,root,,);
+connect (con2,localhost,root,,);
```

Comment:
I believe, you do not need 3 connections (`default`, `con1` and `con2`) in this test case - just `default` and `con1` should suffice.